terms.html

```diff
@@ -168,9 +180,11 @@
   </dd>
   <dt><dfn data-lt="verifier|verifiers|verifier's|credential verifiers|credential verifier's">verifier</dfn></dt>
   <dd>
-A role an <a>entity</a> performs by receiving one or more
-<a>verifiable credentials</a>, optionally inside a
-<a>verifiable presentation</a> for processing. Other specifications might refer
+A role an <a>entity</a> performs by requesting for, and receiving one or more
+<a>verifiable credentials</a> that are authored by <a>entities</a> that it trusts,
```

Comment:
```suggestion
<a>verifiable credentials</a> from any issuer,
```

There is no  requirement that a holder trust the entity issuing the VC. In fact, in many situations, the user has no choice about whether or not they trust the issuer. You're going to get your driver's license from the DMV and you truly don't need to "trust" the DMV any more than you can throw them.